pkg/cover/html.go

```diff
@@ -445,10 +445,29 @@ func groupCoverByFilePrefixes(datas []fileStats, subsystems []mgrconfig.Subsyste
 		var percentCoveredFunc float64
 
 		for _, path := range subsystem.Paths {
+			if strings.HasPrefix(path, "-") {
+				continue
+			}
+			var excludes []string
+			for _, exclude := range subsystem.Paths {
+				if strings.HasPrefix(exclude, "-") && strings.HasPrefix(exclude[1:], path) {
+					excludes = append(excludes, exclude[1:])
+				}
+			}
 			for _, data := range datas {
 				if !strings.HasPrefix(data.Name, path) {
 					continue
 				}
+				excluded := false
```

Comment:
This code also deserves to be a helper function. As a positive side effect, you will not need the `break` and `excluded` variable. Just `return`.